CONTRIBUTION.md

```diff
@@ -16,12 +16,20 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 ### Setup Project for Development and Testing
 
-1. Follow the package setup instructions on the main README till the [`Clone Project Repository`](README.m#131-clone-project-repository) step.
-2. If you are a `schematic` package contributor you don't need to create a `venv` virtual environment, `poetry` will create a virtual environment by default, which you can use.
-3. Running the following command reads the [`pyproject.toml`](https://github.com/Sage-Bionetworks/schematic/blob/develop/pyproject.toml) file from the current project, resolves the dependencies and installs them: `poetry install`
-4. Obtain credentials file(s) by following same instructions as in the [`1.3.4. Obtain Credentials File(s)`](README.md#134-obtain-credentials-files) section.
-5. Fill in configuration file(s) in the same way as specified in the [`1.3.5. Fill in Configuration File(s)`](README.md#135-fill-in-configuration-files).
-6. To run any of the CLI utilities shown in the [`1.3.6. Command Line Interface`](README.md#136-command-line-interface), prefix the commands with `poetry run`.
+1. Install [package dependencies](https://sage-schematic.readthedocs.io/en/develop/README.html#installation-requirements-and-pre-requisites).
+2. Clone the `schematic` package repository: `git clone https://github.com/Sage-Bionetworks/schematic.git`
+3. [Create and activate](https://sage-schematic.readthedocs.io/en/develop/README.html#virtual-environment-setup) a virtual environment.
+4. Run the following commands to build schematic and install the package along with all of its dependencies:
+   ```python
+   cd schematic  # change directory to schematic
+   git checkout develop  # switch to develop branch of schematic
+   poetry build # build source and wheel archives
+   pip install dist/schematicpy-0.1.11-py3-none-any.whl  # install wheel file
```

Comment:
Another set of triple-backticks missing?

Sorry, I'm reading through this file in reverse for some reason.